src/SeqCli/Cli/Commands/ApiKey/ListCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SeqCli.Cli.Features;
+using SeqCli.Connection;
+using Serilog;
+
+namespace SeqCli.Cli.Commands.ApiKey
+{
+    [Command("apikey", "list", "Send a structured log event to the server", Example =

```

Comment:
The description of the command should be updated here :)

Reply:
Thanks @tsimbalar I have fixed that up now.